frontend/apps/marketing/tests/e2e/all-the-things.spec.ts

```diff
@@ -25,7 +25,7 @@ test.describe('All the things UI e2e test', () => {
           JSON.stringify(accessibilityScanResults.violations, null, 2),
         );
 
-        expect(accessibilityScanResults.violations.length).toEqual(2);
+        expect(accessibilityScanResults.violations.length).toEqual(1);
```

Comment:
Updated this to fix a failing ui test introduced [here](https://github.com/code-dot-org/code-dot-org/pull/65962/files).